semcore/d3-chart/src/component/ChartLegend/LegendFlex/LegendFlex.type.ts

```diff
@@ -0,0 +1,38 @@
+import { Flex, FlexProps } from '@semcore/flex-box';
+import { Intergalactic } from '@semcore/core';
+import { LegendItemType } from '../LegendItem/LegendItem.type';
+import { LegendProps } from '../BaseLegend.type';
+
+export type TrendProps = {
+  /**
+   * Flag for include trend LegendItem in Legend
+   */
+  withTrend: true;
+  /**
+   * Trend label (for localization)
+   */
+  trendLabel: string;
+  /**
+   * Flag for describe are trend is visible
+   */
+  trendIsVisible: boolean;
+  /**
+   * Handler for change trend visibility
+   */
+  onChangeTrendVisible: (isVisible: boolean) => void;
```

Comment:
Probably it should be named `onTrendIsVisibleChange`?

Reply:
yes, i think it will be better